client/i18n/locales/english/intro.json

```
@@ -2093,7 +2093,12 @@
       "ybrh": { "title": "271", "intro": [] },
       "vjgg": { "title": "272", "intro": [] },
       "kaui": { "title": "273", "intro": [] },
-      "afhj": { "title": "274", "intro": [] },
+      "lab-nth-fibonacci-number-generator": {
+        "title": "Build the nth Fibonacci number generator",
```

```suggestion
        "title": "Build an nth Fibonacci number generator",
```
I would wait for a native speaker to confirm that it sounds right in one way or another.

---

then there is alto to add an en-dash after `nth`, nth–Fibonacci number, maybe?